go/vt/vtadmin/rbac/authentication.go

```diff
@@ -99,7 +99,13 @@ type actorkey struct{}
 // NewContext returns a context with the given actor stored in it. This is used
 // to pass actor information from the authentication middleware and interceptors
 // to the actual vtadmin api methods.
+//
+// If actor is nil, the context is returned with no actor attached.
 func NewContext(ctx context.Context, actor *Actor) context.Context {
+	if actor == nil {
+		return ctx
+	}
```

Comment:
this is fine (and agreed, nicer), but in general i dont think we should worry too much about "violations" of analysis tools in generated code (esp generated code in tests)

Reply:
Yep that's totally fair. I made the change because it wasn't particularly intrusive and cleaned up a lint for what might be a reasonably common use case anyway.